android/java/org/chromium/chrome/browser/crypto_wallet/util/AssetUtils.java

```diff
@@ -56,4 +57,21 @@ public static boolean isAuroraAddress(String contractAddress, String chainId) {
         return (isEthereumBridgeAddress || isNativeAsset)
                 && chainId.equals(BraveWalletConstants.MAINNET_CHAIN_ID);
     }
+
+    public static @CoinType.EnumType int getCoinForKeyring(String keyringId) {
```

Comment:
please put it into Utils.java where the backwards conversion is done https://github.com/brave/brave-core/blob/1.46.x/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java#L1570 or move the one from Utils.java to AssetUtils.java. It doesn't look good that two related helper methods are in separate files.

Reply:
I would avoid to moving this method to Utils.java to avoid adding more work for [utils refactoring ticket](https://github.com/brave/brave-browser/issues/20028) so will move the existing implementation to AsserUtils.